src/libs/OptionsListUtils.js

```diff
@@ -82,9 +82,16 @@ function createOption(personalDetailList, report, draftComments, activeReportID)
         && (report.reportID !== activeReportID)
         && lodashGet(draftComments, `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${report.reportID}`, '').length > 0;
 
+    const lastActorDetails = report ? personalDetailList.find(({login}) => login === report.lastActorEmail) : null;
     return {
         text: report ? report.reportName : personalDetail.displayName,
-        alternateText: (report && hasMultipleParticipants) ? report.reportName : personalDetail.login,
```

Comment:
Let's continue to use `alternateText` for this instead of changing the name. The options could be used by other things in the future.